### ISSUE TITLE
Getting "Fatal error: Uncaught Error: Undefined class constant 'PROFILE_DATA_OPTION'" when installing or upgrading to WooCommerce 6.5 with an active WooCommerce Admin plugin version 3.2.1 or lower 

### ISSUE THREAD

**Issue by rodelgc** [woothemes/woocommerce#32962]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When you upgrade to WooCommerce 6.5 while having an activated WooCommerce Admin plugin with version 3.2.1 or lower, you'll get the error `Fatal error: Uncaught Error: Undefined class constant 'PROFILE_DATA_OPTION'`

### Expected behavior

1. No fatal error.
2. Warning message saying `WooCommerce Admin plugin has been deactivated to avoid conflicts with WooCommerce plugin` is shown after the upgrade.
3. WooCommerce Admin plugin is automatically deactivated.

### Actual behavior

Fatal error was displayed.

### Steps to reproduce

1. Install and activate WooCommerce 6.4.1 or any version lower than 6.5.
2. Install and activate WooCommerce Admin 3.2.1 or lower
3. Upgrade WooCommerce to version 6.5.0.
4. Notice the fatal error.

Screenshot: p1652274052401699/1652268042.762609-slack-C01DT6U03HC

### WordPress Environment

WooCommerce: 6.4.1
WooCommerce Admin: 3.2.1

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Comment by beaulebens** [comment]
Note this is confirmed and considered serious enough for us to pause roll-out of WC 6.5. We're working on a fix and will release 6.5.1 to address it shortly.

**Comment by JoshuaGoode** [comment]
Re: p1652281880343069-slack-C7U3Y3VMY

p1652294916621039-slack-CPJE78JQ5

**Comment by amElnagdy** [comment]
5206168-zen

**Comment by Ajanthini** [comment]
I have updated latest version 7.6.0 of WooCommerce. After updating WooCommerce I am getting error saying "Database
Missing base tables: wp_woocommerce_order_itemmeta. Some WooCommerce functionality may not work as expected."

I went to "phpmyadmin" I checked that table but I couldn't opened too. I am getting an error saying that "wp_woocommerce_order_itemmeta" is marked as crashed and last(automatic?) repair failed.

How to fix it?
